docs/syzbot_assets.md

```diff
@@ -0,0 +1,174 @@
+## Reproduce a bug with syzbot's downloadable assets
+
+As a part of every bug report, syzbot has begun to share downloadable assets --
+that is, disk images and kernel binaries on which the bug was originally found.
```

Comment:
You formulated it as a news.
"syzbot shares downloadable assets" will always be up-to-date.